releasenotes/notes/configcheck-print-not-reporting-msg-c7d2ff2a730d09a1.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+enhancements:
+  - |
+    `configcheck` prints a message for checks that are not reporting metrics or
+    logs.
```

Comment:
This might sounds like the message is printed when the number of metric samples or logs is not increasing. To avoid this confusion, wdyt about:
```
The ``agent configcheck`` command prints a message for checks that matched a container exclusion rule.
```
(Note: We need to use double quotes `` instead of ` in releasenotes.)

Reply:
Fixed :+1: